### PR TITLE
docs: update the quick start WSL info

### PR DESCRIPTION
**Pull request by firebomb1**
## Description

Updating the README with more details about the Windows Subsystem for Linux setup.
Changing the order of commands/descriptions to put commands first for more readability.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/blob/develop/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)
N/A

## Risk

None

## Testing
N/A

### Engineering
N/A

### Operations
N/A

## Screenshots (if applicable)
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,35 +47,40 @@ ShapeShift's OSS 2nd generation Web application. (Under Development)
 
 ## Quick Start
 
-If you are using Linux and MacOS it works out of the box following the steps.<br/>
-⚠️ On Windows you should use the _Windows Subsystem for Linux_ (WSL).
+If you are using Linux and MacOS it works out of the box following the steps.
 
-- Clone the repo
+> ⚠️ On Windows you should use the _Windows Subsystem for Linux_ (WSL) and make sure to use the WSL filesystem to store your clone of the repository in order to avoid issues with line endings and enjoy better performance.
```
Comment (gomesalexandre) [inline README.md:52]: `_Windows Subsystem for Linux_ (WSL)` should be a link?

**Review by gomesalexandre — COMMENTED** [review]
Looks good @firebomb1 , however I'm not sure we want to get this in with the WSL addition.

tl;dr is there are a lot of ramifications WRT working with our codebases on non UNIX/POSIX compliant systems and hinting that you can contribute as a WSL user (which I'm sure can be done, as WSL works for the most part) might effectively make engineering WSL support, which is something we want to avoid.

I'll let the team comment on that, but I'd personally vouch for reverting the WSL part and keep this PR to the reorder of steps, which looks good to me

File: README.md (hunk 1 above)
Comment (firebomb1) [reply README.md:52]: We could also mention in the same paragraph that no additional support will be provided for developers using WSL, because our focus is to support dev environments for Linux/Mac OS.

From what I read WSL2 is already a big improvement in an attempt to match a native Linux distro (using a VM for storage with an EXT4 file system). But obviously it's [unlikely to ever match 100%](https://learn.microsoft.com/en-us/windows/wsl/compare-versions) but they already claim "100% system call compatibility" apparently.

I do expect "hdwallet" which deals with WebUSB to be much more problematic with WSL (just reading the linked limitations above, but I have not tested it). My suggestion to explicitly limit the support would at least not discourage potential future contributors who just want to first test our code and experiment.

Comment (0xApotheosis) [reply README.md:52]: Yeah, agree with where your head is at @gomesalexandre. Maybe to @firebomb1's suggestion we want to add language stating very clearly that we do not support approach this in any way, and so do not ask the engineering team about it - but if you want to play around with it good luck.

Comment (gomesalexandre) [reply README.md:52]: Yeah, as long as we add a warning alongside the instructions, I'm fine with it @0xApotheosis

Comment (firebomb1) [reply README.md]: Thanks, the requested changes have been applied. The Windows section has been moved at the end of quick start guide to also signify it isn't something we intend to support.

**Review by 0xApotheosis — CHANGES_REQUESTED** [review]
I like the addition re: the WSL comments 👌 

I'm not sure about the re-ordering of the comments and descriptions.
The confusion might be because our tense is wrong. It should really be declarative, e.g.

Do a thing:

`here is the code that does the thing`

An example from the [aws-cli](https://github.com/aws/aws-cli) docs:

Screenshot 2022-11-16 at 4 53 48 pm

If you don't want to re-write it just revert those changes and we can keep the WSL addition 🙏

File: README.md [hunk 2]
```diff
@@ -47,80 +47,94 @@ ShapeShift's OSS 2nd generation Web application. (Under Development)
...
-    yarn dev
+yarn dev
 ```
 
-Launches the test runner in the interactive watch mode.<br /> See the section
-about
-[running tests](https://facebook.github.io/create-react-app/docs/running-tests)
-for more information.
-It also creates a html page you can interact with at the root level of the project in `/coverage`.
+> It opens [http://localhost:3000](http://localhost:3000) to view it in the browser and the page will reload if you make edits.
```
Comment (gomesalexandre) [inline README.md:83]: Very nitpicky but `this` wordage instead of `it` (referring to the aforementioned command) seems more natural, but this is already an improvement as is

**Review by gomesalexandre — APPROVED** [review]
Both the new WSL wordage with warnings and instructions verbiage and reorder LGTM now, thanks for this one! 🏅 
Added a smallish comment on how to improve it, feel free to tackle at your own discretion.

File: README.md (hunk 2 above)
Comment (gomesalexandre) [reply README.md:83]: Alternatively, ditching `it` and keeping the verb in present tense also works
